runtime/msg.c

```diff
@@ -656,6 +658,8 @@ propNameToID(uchar *pName, propid_t *pPropID)
 		*pPropID = PROP_SYS_QHOUR_UTC;
 	} else if(!strcasecmp((char*) pName, "$minute-utc")) {
 		*pPropID = PROP_SYS_MINUTE_UTC;
+	} else if(!strcasecmp((char*) pName, "$wday-utc")) {
```

Comment:
functional enhancement (series) -- needs doc update

Reply:
I know and I will ;-)